source/_drafts/Autoplay-Best-Practices-with-Video-js.md

```diff
@@ -0,0 +1,113 @@
+---
+title: "Autoplay Best Practices with Video.js"
+author:
+  name: "Pat O'Neill"
+  github: misteroneill
+tags:
+---
+
```

Comment:
Thoughts about having a `tl;dr` section that basically says "use player.play() for autoplaying" before the post?

Reply:
Sounds good.